h/static/scripts/annotator/plugin/enhancedanchoring.coffee

```diff
@@ -83,6 +89,37 @@ class Anchor
       # Kill the list if it's empty
       delete @anchoring.anchors[index] unless anchors.length
 
+  # Scroll to this anchor
+  scrollTo: ->
```

Comment:
> Why is this called scrollTo, but it takes no arguments? Perhaps take a tip from our jQuery plugin and call it `scrollIntoView`?

Great idea. Rename everywhere in https://github.com/hypothesis/h/commit/3606994ed7b1f3f3a38a44b31051f32b4bcbd922.